src/base/QXmppRtpChannel.cpp

```diff
@@ -490,9 +490,7 @@ qint64 QXmppRtpAudioChannel::readData(char * data, qint64 maxSize)
 void QXmppRtpAudioChannel::payloadTypesChanged()
 {
     // delete incoming codecs
-    foreach (QXmppCodec *codec, d->incomingCodecs)
-        delete codec;
-    d->incomingCodecs.clear();
+    qDeleteAll(d->incomingCodecs);
```

Comment:
Hey! You forgot to call `clear()` here. Please re-read `qDeleteAll()` usage in documentation.

Reply:
Quite frankly I feel like this code is just asking for `std::unique_ptr`s. Manual memory management, explicit deletes, eww... But this may be a task for another day and another PR.

Reply:
Sorry I should have read the documentation of `qDeleteAll()` first.